internal/coreinternal/attraction/attraction_test.go

```diff
@@ -86,7 +86,7 @@ func TestAttributes_InsertValue(t *testing.T) {
 		},
 	}
 
-	ap, err := NewAttrProc(cfg)
+	ap, err := NewAttrProc(cfg, nil)
```

Comment:
I won't block the PR for this unless @pmm-sumo agrees with me, but I would prefer the test to be passing zap.NewNop() instead of nil. Remember that your test code shows the good practice for consumers of the main code, and by passing nil, you are sending the message that nil is OK, when it really isn't.

Reply:
Sure, I don't have a preference here. Gonna update.